acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -17,14 +17,16 @@
 public class NodeConfiguration {
 
   private static final String HTTP_URL_FORMAT = "http://%s:%s";
-  private static final int TCP_PORT = 8545;
-  private static final int WS_PORT = 8546;
+  private static final int TCP_PORT = 8547;
+  private static final int WS_PORT = 8548;
   private static final Duration POLLING_INTERVAL = Duration.ofMillis(500);
 
   private final String hostname;
+  private final String genesisFile;
 
-  public NodeConfiguration(final String hostname) {
+  public NodeConfiguration(final String genesisFile, final String hostname) {
```

Comment:
nit: rename to genesisFilename

Reply:
Renaming, but to `genesisFilePath`, as filename could mean the name of the file, however this is the entire path

Reply:
that sounds even smarter.